### PR TITLE
57@muyah

### PR DESCRIPTION
**Pull request by dmuyah**
## Fixes Issue 
 `Closes#57`

## Changes proposed

- [x]  Corrected #57 

- [X]  Added space on Landing page info
- [X]  Added emoji and animation on emoji and logo on hover

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -8,17 +8,28 @@ Please note we have a [code of conduct](https://github.com/reactdeveloperske/rea
 
 - Create a [new issue](https://github.com/reactdeveloperske/reactdevske-website/issues)
 - Comment on the issue (if you'd like to be assigned to it) - that way the issue can be assigned to you.
+  
+### Open issues
+- Go to [open issues](https://github.com/reactdeveloperske/reactdevske-website/issues)
+- select issue of your choice that is `open` or `need help` and is not assigned.
```
Comment (Lekipising) [inline CONTRIBUTING.md:14]: Suggested edits:

Change to `Select an issue...`

Also they should check if the user has `up for grabs` label

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -49,37 +60,47 @@ $ git merge upstream/develop
...
+<br>
+
+### Start developing! 
+<br>
+
 - Open this directory in your favorite text editor / IDE, and see your changes live by visiting `localhost:3000` from your browser
 - Pro Tip: Explore scripts within `package.json` for more build options
 
-3. Commit and prepare for pull request (PR). In your PR commit message, reference the issue it resolves (see [how to link a commit message to an issue using a keyword](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
+1. Commit and prepare for pull request (PR). In your PR commit message, reference the issue it resolves (see [how to link a commit message to an issue using a keyword](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
```
Comment (Lekipising) [inline CONTRIBUTING.md:103]: `PR commit message` may give a wrong impression. Suggesting `PR description`

**Comment by Lekipising** [comment]
Suggesting a more descriptive PR title.

**Comment by orama254** [comment]
Hello @dmuyah 

thank you for this pull request

from issue #57 I can see that the only change that was supposed to be made was in regards to the commands on the contribution guidelines.

for tracking purposes it's advisable to make one pull request with related commits per issue. I can see you also made some commits where you made changes on the codebase as well.

Kindly check on this and I'll be happy to review

File: components/Navbar.jsx [hunk 3]
```diff
@@ -18,10 +18,9 @@ const Navbar = () => {
       <header className="px-[181px] w-full shadow-md bg-white">
         <nav className="nav py-4 flex justify-between items-center">
           <div>
-            <Link href={HOME} >
-              <a >
-              <Image src={logo} className="hover:animate-spin" alt="reactdevske logo" width={137} height={60} />
-              </a>
```
Comment (orama254) [inline components/Navbar.jsx:24]: there is this closing a tag that's missing from your commit

Comment (dmuyah) [reply components/Navbar.jsx:24]: corrected sorry for that

**Comment by dmuyah** [comment]
> Hello @dmuyah
> 
> thank you for this pull request
> 
> from issue #57 I can see that the only change that was supposed to be made was in regards to the commands on the contribution guidelines.
> 
> for tracking purposes it's advisable to make one pull request with related commits per issue. I can see you also made some commits where you made changes on the codebase as well.
> 
> Kindly check on this and I'll be happy to review

**Pull request merged**